src/main/java/cloud/fogbow/ms/Main.java

```diff
@@ -26,8 +28,11 @@ public void run(ApplicationArguments args) throws ConfigurationErrorException {
             ServiceAsymmetricKeysHolder.getInstance().setPublicKeyFilePath(publicKeyFilePath);
             ServiceAsymmetricKeysHolder.getInstance().setPrivateKeyFilePath(privateKeyFilePath);
             
+            AuthorizationPlugin<AdminOperation> authorizationPlugin = PluginInstantiator.getAuthorizationPlugin();
```

Comment:
I think the class name should be MsOperation, instead of AdminOperation, to be consistent with the other services.

Reply:
Done.